.gitignore

```diff
@@ -1 +1,130 @@
 *.swp
+# ---> Go
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
+
+# ---> JetBrains
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and WebStorm
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff
+.idea
```

Comment:
This file should contain repository-specific ignore patterns. Please use global ignore file for editors, etc: https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
@yiranzai to configure a global Gitignore, here is an example:

```bash
mkdir -p ~/.config/git
curl -s \
  https://raw.githubusercontent.com/github/gitignore/master/{Global/JetBrains,Global/Vim,Global/VisualStudioCode,Global/macOS,Python,Terraform,Node}.gitignore \
  > ~/.config/git/ignore
```

Then try reverting this `.gitignore`'s changes and seeing what remains in the diff